README.md

```diff
@@ -19,8 +19,9 @@ Testing
 
 We run the tests using `tox`. This can be installed as usual with `pip install tox`.
 
-Without any options, `tox` will run the tests against the current python version where
-`tox` itself was installed:
+Without any options, `tox` will run the tests against all of the library's
+target Python versions, which will cause an error if one of those versions is
```

Comment:
you mean it does _not_ cause an error...

Reply:
oops, yes, I forgot to change that